pallets/delegation/src/lib.rs

```diff
@@ -231,16 +278,17 @@ pub mod pallet {
 		pub fn add_delegation(
 			origin: OriginFor<T>,
 			delegation_id: DelegationNodeIdOf<T>,
-			root_id: DelegationNodeIdOf<T>,
-			parent_id: Option<DelegationNodeIdOf<T>>,
+			root_node_id: DelegationNodeIdOf<T>,
+			parent_id: DelegationNodeIdOf<T>,
```

Comment:
would it be possible to pass a different `root_node_id` than the one that is stored in the parent? Since we always query the parent,  no need to have the root node as a parameter?

Reply:
Addressed in https://github.com/KILTprotocol/mashnet-node/commit/cbc29eb171e169aa6cc223b017b57d2d7b4303bd.